test/lib/goal_form_test.rb

```diff
@@ -0,0 +1,98 @@
+require 'test_helper'
+
+class GoalFormTest < ActiveSupport::TestCase
+  test 'a simple draft plan' do
+    crosswalk = JSON.load File.open './app/fixtures/crosswalk.json'
```

Comment:
thought of another thing: `crosswalk` and `benchmarks` can be made into instance variables and intialized in the `setup` method:
```
def setup
  @crosswalk = ...
  @benchmarks = ...
end
```
they'll then be accessible to all the `test` blocks.